### ISSUE TITLE
Improve filters for timeline graphs (summary)

### ISSUE THREAD

**Issue by mbrysa** [abrt/faf#443]
Filter by:
* problem type
* kernel modules

**Comment by jfilak** [comment]
I do not think you need the "problem type" filter, you should implement the "per component statistics" (#156) instead.
